packages/search-token-lambda/README.md

```diff
@@ -0,0 +1,29 @@
+# Simple search token generation lamba function
```

Comment:
@jpmarceau Need a review mainly for this file, the rest of the doc is copied from the package/search-token-server in the same repo